src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -850,9 +877,10 @@ GraphPatternOperation Visitor::visit(Parser::ServiceGraphPatternContext* ctx) {
   auto serviceIri =
       TripleComponent::Iri::fromIriref(std::get<Iri>(varOrIri).iri());
 
-  if (serviceIri.toStringRepresentation() ==
-      "<https://qlever.cs.uni-freiburg.de/pathSearch/>") {
+  if (serviceIri.toStringRepresentation() == PATH_SEARCH_IRI) {
     return visitPathQuery(ctx);
+  } else if (serviceIri.toStringRepresentation() == SPATIAL_SEARCH_IRI) {
+    return visitSpatialQuery(ctx);
```

Comment:
If those are the only usages of these Constants, then they shouldn't be inside `Constants.h`, but they should be in a separate header (BuiltinServiceIriConstants.h or something), because `Constants.h` is already big enough.